nylas/client/restful_model_collection.py

```diff
@@ -22,19 +22,26 @@ def __iter__(self):
         return self.values()
 
     def values(self):
+        limit = self.filters.get("limit")
```

Comment:
nitpick: what do you think about using bracket notation or updating the following line to use `.get` too to keep consistent?

Reply:
I'd rather not do that, actually. `.get()` can return a value or `None`, while indexing (using bracket notation) will _always_ return a value (or raise an exception). Since we do `filters.setdefault("offset", offset)` on line 15, there is guaranteed to be a value for `filters["offset"]`, but there is _not_ guaranteed to be a value for `filters["limit"]`. That difference is important, and the difference between `.get()` and indexing is designed to highlight it.

Alternatively, we could make that difference go away, by removing the `.setdefault()` call on line 15. That might even be a better option, because it preserves the fact that the model collection was created without a defined limit. However, that would be a larger change, and I feel like it doesn't belong in this pull request.

Reply:
`__init__` defaults `offset` to 0 so I think keeping `filters.setdefault("offset", offset)` on line 15 makes sense. It might make even more sense to remove the `offset` param completely from `init` and have it passed in as a part of filter. That way, we could remove the `set_default`. In any case, that doesn't belong in this PR either.

This is fine to me as is!